Airport/Program.cs

```diff
@@ -0,0 +1,577 @@
+            Console.WriteLine("Type date:");
+            var date = Console.ReadLine();
+            Console.WriteLine("Type time:");
+            var time = Console.ReadLine();
+            Console.WriteLine("Type price:");
+            var price = double.Parse(Console.ReadLine());
+            Console.WriteLine("Type number of seats:");
+            var quantity = int.Parse(Console.ReadLine());
+            flights.Add(new Flights { Id = id, FlightId = rnd.Next(20, 30), City = city, Country = country, Date = date, Time = time, Price = price, NumberOfSeats = quantity });
+            string json = JsonSerializer.Serialize(flights);
+            var fileName1 = $"{jsonPath}flights.json";
+            File.WriteAllText(fileName1, json);
+            Console.WriteLine("Flight has been succefully added!");
+        }
+        public static void ShowAllFlights()
+        {
+            Console.WriteLine("--------------------------------------------------------");
+            foreach (var item in flights)
+            {
+                if (id == item.Id)
+                {
+                    Console.WriteLine($"|{item.FlightId} | {item.City} | {item.Country} | {item.Date} | {item.Time} | {item.Price} | {item.NumberOfSeats}|");
+                }
+            }
+            Console.WriteLine("--------------------------------------------------------");
+        }
+
+        public static void ShowAllPassengers()
+        {
+            Console.WriteLine("--------------------------------------------------------");
+            int count = 0;
+            for (int i = 0; i < passengers.Count; i++)
+            {
+                for (int j = 0; j < passengers[i].reservation.Count; j++)
+                {
+                    for (int k = 0; k < flights.Count; k++)
+                    {
+                        if (id == flights[k].Id && flights[k].FlightId == passengers[i].reservation[j].FlightId)
+                        {
+                            if (count > 1)
+                            {
+                                Console.WriteLine($"{passengers[i].reservation[j].ReservationId}-{flights[k].City}-{flights[k].Country}-{flights[k].Date}");
+                                Console.WriteLine("****************************");
+                            }
+                            else
+                            {
+                                Console.WriteLine($"{passengers[i].PassengerId}-{passengers[i].FirstName}-{passengers[i].LastName}");
+                                Console.WriteLine("****************************");
+                                Console.WriteLine($"{passengers[i].reservation[j].ReservationId}-{flights[k].City}-{flights[k].Country}-{flights[k].Date}");
+                                count++;
+                            }
+                        }
+                    }
+                }
+            }
+            Console.WriteLine("--------------------------------------------------------");
```

Comment:
this thing can be created like `Enumerable.Range(0, {yourLength}).SelectMany(_ => '_');` and stored like readonly variable